app/views/sales/_form.html.erb

```diff
@@ -0,0 +1,102 @@
+<div class='row'>
+  <div class='col-lg-12'>
+    <%= simple_form_for @sale do |f| %>
+      <%= f.error_notification %>
+
+      <section class='vehicle-fields'>
+        <%= f.simple_fields_for :vehicle_attributes, @sale.vehicle do |fv| %>
+          <div class='form-inputs'>
```

Comment:
Try out using `fieldset` and `legend` too, they have a special usage for blind people (as far as I know).